frontend/src/components/NewProject/ConflictModal.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useContext, useEffect } from 'react'
+import { Button, Form, Input, Modal } from 'semantic-ui-react'
+
+import UseForm from '@hooks/useForm'
```

Comment:
I think this should be lowercase `useForm`?

Reply:
yes